### DISCUSSION TITLE
updating Windows Domain Functional Level

### DISCUSSION THREAD

**Discussion by matiasmg82** [zitadel/zitadel#8332] [Q&A]
Hi all, I've zitadel v2.23 deployed in windows server, i'm concern because of an windows server domain upgrade we're going from a W 2012 R2 to W 2022 and we're going to a functional level change.
Is there any problem with Zitadel if this windows functionality level changed? i've checked all documentation and is nothing for this.
Thanks

**Comment by matiasmg82** [comment]
no

**Comment by fforootd** [comment]
> Hi all, I've zitadel v2.23 deployed in windows server, i'm concern because of an windows server domain upgrade we're going from a W 2012 R2 to W 2022 and we're going to a functional level change. Is there any problem with Zitadel if this windows functionality level changed? i've checked all documentation and is nothing for this. Thanks

Hey that is a fairly old version, so it is hard to tell.

Be aware of the update 2.39.x since that has as important change to it https://zitadel.com/docs/support/advisory/a10004
